api/openapi/v1/support-notifications.yaml

```diff
@@ -46,22 +55,21 @@ paths:
                 $ref: '#/components/schemas/Error'
   /v1/notification:
     post:
-      description: Receive Alerts or Notifications.  If the severity is CRITICAL,
-        the Notifications are processed (distributed) immediately.  Or, the Notification
-        is processed in batch.
+      description: Receive alerts or notification. notification of any severity
+        level are processed/distributed immediately.
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Notification'
+              $ref: '#/components/schemas/notification'
         required: true
       responses:
         202:
-          description: If Alerts and Notifications Service is available, it always
-            returns 202 Accepted to clients with the new ID to indicate the Notification
-            has been received.
+          description: If the alerts and notification service is available, it
```

Comment:
"are available"

Reply:
Incorporated your comments (all _except_ this one, because the `Alerts and Notifications Service` is a single service, not two distinct services). Thorough feedback overall 👍